### PR TITLE
Update crawler.py

### PR DESCRIPTION
**Pull request by jsink13**
its a bit rude to write data by default to the CODE path.  This allows us to pass in a directory when we make the object.

I hope this is OK.

### CODE REVIEW

File: SECEdgar/crawler.py [hunk 1]
```diff
@@ -10,14 +10,15 @@
 
 
 class SecCrawler():
-
-    def __init__(self):
+     
+    def __init__(self, output_directory = None):
         self.hello = "Welcome to Sec Cralwer!"
-        print("Path of the directory where data will be saved: " + DEFAULT_DATA_PATH)
```
Comment (rahulrrixe) [inline SECEdgar/crawler.py]: We can have a check here

```python
     if output_directory not None:
          print("Path of the directory where data will be saved: " + self.output)
     else:
         print("Path of the directory where data will be saved: " + DEFAULT_DATA_PATH)
    
   self.output = output_directory or DEFAULT_DATA_PATH  

```

**Comment by jackmoody11** [comment]
This is addressed in #36. Setting `DEFAULT_DATA_PATH` as the default parameter will help with keeping the code concise (no if else statements).

**Comment by coyo8** [comment]
There is a conflict.

**Comment by coyo8** [comment]
I think; this is no longer required and so I am closing it.

**Pull request closed without merging**